api/src/redis_cache.rs

```diff
@@ -22,13 +22,22 @@ pub async fn warm_validators(
 ) -> anyhow::Result<()> {
     info!("Loading validators from DB");
     let warmup_timer = Instant::now();
-    let validators = store::utils::load_validators(
+
+    let bonds = store::utils::fetch_bonds(&context.read().await.bonds_url).await?;
+    let mut validators = store::utils::load_validators(
         &context.read().await.psql_client,
         scoring_url,
         DEFAULT_EPOCHS,
         DEFAULT_COMPUTING_EPOCHS,
     )
     .await?;
+
+    for bond in bonds {
+        if let Some(validator) = validators.get_mut(&bond.vote_account) {
+            validator.self_stake += bond.effective_amount;
```

Comment:
We should use the `funded_amount` for this purpose, maybe?
My understanding is that the `effective_amount` is calculated from `funded_amount` substracted by the amount that is being to be withdrawn and maybe that which is part of the settlements. Despite the amounts are not "fully active" in bonds for a particular validator stake accounts are attached to the validator and generate some revenue for him.